interface/i18n/mist.nl.i18n.json

```diff
@@ -89,15 +111,15 @@
             "blockReceived": "Nieuw blok ontvangen",
             "blockNumber": "Jouw laatste bloknummer",
             "timeSinceBlock": "Verstreken tijd sinds laatste blok",
-            "testnetExplain": "Je bent actief op een testnet, VERSTUUR GEEN echte Ether naar deze adressen",
-            "peers": "peers",
             "checkingWhichNetwork": "Netwerk aan het controleren...",
             "mainNetwork": "Main-net",
             "testNetwork": "Test-net",
             "privateNetwork": "Private-net",
             "mainnetExplain": "Je zit op het main Ethereum wereldwijde netwerk",
+            "testnetExplain": "Je bent actief op een testnet, VERSTUUR GEEN echte Ether naar deze adressen",
```

Comment:
The English transaction uses the `__name__` placeholder.
Would be nice if you follow the same pattern.